File: CPP/Function/04.student&markproblem.cpp
//Write a Function to Solve Student and Marks Problem

#include <iostream>

using namespace std;


char findGrades(int marks){
    //considering the marks of student lies between 0 and 100
    
    switch (marks/10){
        case 10: 
        case 9: return 'A'; break;
        case 8: return 'B'; break;
        case 7: return 'C'; break;
        case 6: return 'D'; break;
        default: return 'E'; break;
    }
}

int main()
{
    
    int a;
    cout<<"Enter the Marks: ";
    cin>>a;
    

    cout<<"Grade of this student is: "<< findGrades(a);
    
    return 0;
}

File: Python/Function/04.student&markproblem.py
#Write a Function to Solve Student and Marks Problem

marks = int(input("Enter the marks of students: "))

def findGrades(marks):
    #considering the marks of students is between 0 and 100
    if  marks>=90 and marks<=100:
        return 'A'
    elif marks>=80:
        return 'B'
    elif marks>=70:
        return 'C'
    elif marks>=60:
        return 'D'
    else:
        return 'E'
        
Grade = findGrades(marks)

print("Grade of the student is: ", Grade)